src/Helper.js

```diff
@@ -0,0 +1,105 @@
+    linkedin: "https://www.linkedin.com/in/deanna-will-williams/",
+    gitHubb: "https://github.com/DFWilliams",
+    image:"https://media-exp1.licdn.com/dms/image/C4D03AQGZ7mF2y-CZEw/profile-displayphoto-shrink_200_200/0/1663802530270?e=1672876800&v=beta&t=w9WbjYLpf1MA41tIROJAd64nceHAbF33CtKCmrmX93c"
+  },
+  {
+    id: 7,
+    name: "Luis Ramos",
+    cohort: "Delta Cohort 2022",
+    bio:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+    linkedin: "https://www.linkedin.com/in/luis-ramos1/",
+    gitHubb: "https://github.com/ninerboy81",
+    image: "https://media-exp1.licdn.com/dms/image/D5635AQF_2SlZ8_vTrQ/profile-framedphoto-shrink_100_100/0/1664580063482?e=1667937600&v=beta&t=srjEp2hPJ3uApoT9QlDf1XLu5tiFEu1_LJlqdnFdET8"
+  },
+  {
+    id: 8,
+    name: "Ahmed Al Khafaji",
+    cohort: "Delta Cohort 2022",
+    bio:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+    linkedin: "https://www.linkedin.com/in/ahmed-alkhafaji/",
+    gitHubb: "https://github.com/Ahmedkhafaji",
+    image:"https://media-exp1.licdn.com/dms/image/C4E03AQFkhDgRPXVJcg/profile-displayphoto-shrink_100_100/0/1659028736504?e=1672876800&v=beta&t=cYwg0kCIP__yV4b28J4AhFehTP2r6wM3B7RVYSFv0UY"
+  },
+  {
+    id: 9,
+    name: "Don Yoon",
+    cohort: "Charlie Cohort 2022",
+    bio:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+    linkedin: "https://www.linkedin.com/in/don-yoon/",
+    gitHubb: "https://github.com/don-yoon",
+    image: "https://media-exp1.licdn.com/dms/image/D4D35AQH96E1rJ-jlaw/profile-framedphoto-shrink_100_100/0/1656358233624?e=1667944800&v=beta&t=5Lk8cpKJwMjValEMJwfPSktHJcGAgUTK5Z3QT3X2SFM"
+  },
+  {
+    id: 10,
+    name: "Logan Ramos",
+    cohort: "Charlie Cohort 2022",
+    bio:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+    linkedin: "https://www.linkedin.com/in/loganramos/",
+    gitHubb: "https://github.com/Renzlawr",
+    image: "https://media-exp1.licdn.com/dms/image/D4E35AQGHaeFXbLVpvw/profile-framedphoto-shrink_100_100/0/1663101626179?e=1667944800&v=beta&t=H_eD-dtxc3Jflsp8z5S32-8WW2JRhQtIW233u0GizfU"
+  },
+  {
+    id: 11,
+    name: "Sarah Proctor",
+    cohort: "Delta",
+    bio:"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+    linkedin: "https://www.linkedin.com/in/sarah-proctor-sd/",
+    gitHubb: "https://github.com/sjproctor",
+    image:"https://media-exp1.licdn.com/dms/image/C5603AQG9g28R_jOZVw/profile-displayphoto-shrink_100_100/0/1572932473516?e=1672876800&v=beta&t=azzxybOkIe_Vc4A1826ieDAjnEpAnKEevl7yqvAU79A"
+  }
+
+]
+
```

Comment:
This file could use a general cleanup of spacing and syntax. Make sure there is one space between the key and the value.